catalogue_api/api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayAgent.scala

```diff
@@ -0,0 +1,23 @@
+package uk.ac.wellcome.platform.api.models
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+import uk.ac.wellcome.models.AbstractAgent
+
+@ApiModel(
+  value = "Agent"
+)
+case class DisplayAgent(
+  @ApiModelProperty(
+    value = "The name of the agent"
+  ) label: String,
+  @JsonProperty("type") ontologyType: String = "Agent"
+)
+
+case object DisplayAgent {
```

Comment:
I tried to do regular classes with a distinction between DisplayAgent and DisplayOrganisation, but got bogged down in the tests. I'm proposing we do it like this for now, and if we want to define more specific types, we can do that in a separate patch.

Reply:
👍

Reply:
BTW `Location` is being split into `DigitalLocation` and `PhysicalLocation` in #1462 and I ran into probably similar issues so, if we decide to do the same for `Agent`, we can adopt a similar solution hopefully